src/components/contacts-list/contacts-list.container.js

```diff
@@ -1,13 +1,15 @@
 import { connect } from 'react-redux';
-import { fetchContacts } from '../../actions';
+import { fetchContacts, fetchFavourites } from '../../actions';
 import ContactsList from './contacts-list.component';
 
 const mapStateToProps = state => ({
-  contacts: state.data.contacts
+  contacts: state.data.contacts,
```

Comment:
Could you rename this `data` to something like `contactsData`? Once you'll get more reducers in the app, `data` will not be a suggestive name.
Sorry for skipping this in the last review